exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/ProtobufSerializerTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.common.serialization;
+
+import static com.exonum.binding.common.serialization.StandardSerializersRoundtripTest.roundTripTest;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.exonum.binding.common.serialization.TestProtos.Point;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import org.junit.Rule;
```

Comment:
Can you update it to Junit5 after my PR would be merged. https://github.com/exonum/exonum-java-binding/pull/489